focus-android/app/src/main/java/org/mozilla/focus/settings/advanced/SecretSettingsFragment.kt

```diff
@@ -26,6 +28,12 @@ class SecretSettingsFragment :
 
     override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
         addPreferencesFromResource(R.xml.secret_settings)
+
+        requirePreference<SwitchPreferenceCompat>(R.string.pref_key_remote_server_prod).apply {
+            isVisible = true
+            isChecked = context.settings.remoteSettingsServer
+            onPreferenceChangeListener = SharedPreferenceUpdater()
+        }
     }
 
     override fun onSharedPreferenceChanged(sharedPreferences: SharedPreferences, key: String?) {
```

Comment:
haven't tested this locally. is this function getting called for the new preference in Focus?

Reply:
It is! Yes. Also gonna add screenshots